packages/forma-36-react-components/src/components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import cn from 'classnames';
+import Heading from '../Typography/Heading';
+import Paragraph from '../Typography/Paragraph';
+
+const styles = require('./EmptyState.css');
+
+export type EmptyStateProps = {
+  /**
+   * Class names to be appended to the className prop of the component
+   */
+  className?: string;
+  /**
+   * Child nodes to be rendered in the component
+   */
+  children?: React.ReactNode;
+  /**
+   * An ID used for testing purposes applied as a data attribute (data-test-id)
+   */
+  testId?: string;
+  /**
+   * Props for image block
+   */
+  image?: ImageProps;
```

Comment:
Maybe rename to `imageProps` - we use this convention so that it's clearer what kind of prop this is just from the name